README.md

```diff
@@ -77,8 +85,9 @@ To define a new logging feature:
 
 1. Create a `.jsonnet` file under `features/<your-feature>/`
 2. Run `npm run buildJson` to generate the `.json` file
-3. Add a test under `__tests__/<your-feature>.test.js`
-4. Run `npm test` to validate
+3. Run `npm run buildDiagrams` to generate the `.mmd` file
```

Comment:
Should we also add a command/script that runs everything with single command to simplify?